sys/hoon.hoon

```diff
@@ -674,6 +674,13 @@
     --
   a
 ::
+::  +bake Convert wet gate to dry gate by specifying argument mold
```

Comment:
`+bake: convert wet gate to dry gate by specifying argument mold`

Reply:
ah, I just used a 1-nearest neighbor algorithm for determining comment style.